File: add/codesnippet/CSharp/bcf38c99-fee4-4c35-b0be-_1.cs
            Atom10FeedFormatter feedFormatter = new Atom10FeedFormatter();
            XmlReader atomReader = XmlReader.Create("http://Contoso/Feeds/MyFeed");
            if (feedFormatter.CanRead(atomReader))
            {
                feedFormatter.ReadFrom(atomReader);
                atomReader.Close();
            }

File: add/codesnippet/VisualBasic/bcf38c99-fee4-4c35-b0be-_1.vb
        Dim feedFormatter As New Atom10FeedFormatter()
        Dim atomReader As XmlReader = XmlReader.Create("http://Contoso/Feeds/MyFeed")
        If feedFormatter.CanRead(atomReader) Then
            feedFormatter.ReadFrom(atomReader)
            atomReader.Close()
        End If